packages/client/components/GitHubScopingSearchBar.tsx

```diff
@@ -32,9 +42,11 @@ const GitHubScopingSearchBar = (props: Props) => {
 
   return (
     <SearchBar>
-      <GitHubScopingSearchHistoryToggle meetingRef={meeting} />
-      <GitHubScopingSearchInput meetingRef={meeting} />
-      <GitHubScopingSearchFilterToggle meetingRef={meeting} />
+      <SearchBarWrapper>
```

Comment:
We're reusing a lot of the same code in the Poker scope search bars so I considered creating reusable components. However, I decided not to because the GitHub integration doesn't use the CurrentFilters component, `ScopingSearchHistoryToggle` doesn't exist in the Parabol or GitLab integration, and the data we're getting in each of the components differs quite a bit.